src/features/Product/components/EditingComment.tsx

```diff
@@ -0,0 +1,62 @@
+import { ChangeEvent, useState } from "react";
+import { timeAgo } from "shared/utils/formatDate";
+import AuthorInfo from "./AuthorInfo";
+import Textarea from "shared/components/Textarea";
+import Button from "shared/components/Button";
+import styles from "./EditingComment.module.css";
+
+interface Props {
+  content: string;
+  writer: { nickname: string; image: string };
+  createdAt: string;
+  onCancel: () => void;
```

Comment:
P3:
아무 인자도 받지 않고 반환하지 않는 함수는 아래처럼도 표현할 수 있습니다.

```suggestion
  onCancel: voidFunction;
```